admin-ui/app/index.html

```diff
@@ -98,7 +98,7 @@
                   <span class="pficon pficon-warning-triangle"></span>
                   <span class="pficon pficon-warning-exclamation"></span>
               </span>
-              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})"><strong>Push failed:</strong> {{ warning.applicationName }}</a>
+              <a href ng-link="appDetail({app: warning.applicationID, tab: 'activity'})">Variant <strong>{{ warning.variant.name }}</strong> has push failures.</a>
```

Comment:
Yeah that is the problem , at this point we don't know if there one or more failure and it would not be optimal to made another call just to retrieve that. 
Let's try to find a generic warning label.